vmm/src/cpu.rs

```diff
@@ -2092,6 +2096,12 @@ impl Pausable for CpuManager {
             }
         }
 
+        for state in self.vcpu_states.iter() {
```

Comment:
Please add a comment here to the expected behavior.

Reply:
Done :)